### ISSUE TITLE
Error in init(): daemonsets.extensions \"elasticsearch-operator-sysctl\" already exists

### ISSUE THREAD

**Issue by cyrillk** [upmc-enterprises/elasticsearch-operator#198]
I used to have `elasticsearch-operator` in the cluster some time ago. Today I've tried to deploy it again from the helm chart but it didn't work for me.

### Helm command

```sh
helm install es-operator/elasticsearch-operator --set rbac.enabled=True --namespace storage
```

### Pod logs

__"Error in init(): daemonsets.extensions \"elasticsearch-operator-sysctl\" already exists"__ doesn't make sense as `elasticsearch-operator-sysctl` was created by the chart itself.

```
kubectl logs pod/elasticsearch-operator-59df6c87c7-4k95k -n storage
time="2018-05-10T15:05:34Z" level=info msg="elasticsearch operator starting up!"
time="2018-05-10T15:05:34Z" level=info msg="Using Variables:"
time="2018-05-10T15:05:34Z" level=info msg="   baseImage: upmcenterprises/docker-elasticsearch-kubernetes:6.1.3_0"
time="2018-05-10T15:05:34Z" level=info msg="Using InCluster k8s config"
time="2018-05-10T15:05:34Z" level=info msg="SKIPPING: already exists \"elasticsearchclusters.enterprises.upmc.com\"\n"
time="2018-05-10T15:05:34Z" level=error msg="Error in init(): daemonsets.extensions \"elasticsearch-operator-sysctl\" already exists"
time="2018-05-10T15:05:34Z" level=info msg="Found 0 existing clusters "
time="2018-05-10T15:05:34Z" level=info msg="Watching for elasticsearch events..."
panic: runtime error: slice bounds out of range

goroutine 41 [running]:
github.com/upmc-enterprises/elasticsearch-operator/pkg/processor.(*Processor).processPodEvent(0xc42042c900, 0xc42066e630, 0x0, 0x0)
	/Users/stevesloka/godev/src/github.com/upmc-enterprises/elasticsearch-operator/pkg/processor/processor.go:234 +0x261
github.com/upmc-enterprises/elasticsearch-operator/pkg/processor.(*Processor).WatchDataPodEvents.func1(0xc420058600, 0xc42042c900, 0xc4202e2600, 0xc420058de0, 0xc42037c590)
	/Users/stevesloka/godev/src/github.com/upmc-enterprises/elasticsearch-operator/pkg/processor/processor.go:107 +0x223
created by github.com/upmc-enterprises/elasticsearch-operator/pkg/processor.(*Processor).WatchDataPodEvents
	/Users/stevesloka/godev/src/github.com/upmc-enterprises/elasticsearch-operator/pkg/processor/processor.go:103 +0x78
```

**Comment by tmirks** [comment]
@cyrillk Pretty sure it's because of #197; check to see if the pod is still running in your `default` namespace. This may only affect helm deployments, I'm going to experiment.

**Comment by stevesloka** [comment]
I think we have this resolved with an arg that was addedd in #220. We also have #231 which aims to default this to where the operator is deployed.
